utils/python/update_acme_tests.py

```diff
@@ -48,8 +48,8 @@
                              "ERR.f45_g37_rx1.A",
                              "ERP.f45_g37_rx1.A",
                              "SMS_D_Ln9.f19_g16_rx1.A",
-                             "DAE.f19_f19.A",
-                             "PRE.f19_f19.ADESP")
```

Comment:
Something is weird. f19_f19 is a 2 degree run which I can usually do on a workstation. f45_f45 should be very fast. Which run is hanging?